pom.xml

```diff
@@ -57,6 +57,9 @@
     <json-simple.version>1.1.1</json-simple.version>
     <commons-lang.version>2.6</commons-lang.version>
     <rxjava.version>2.2.3</rxjava.version>
+    <kafka-clients.version>1.0.2</kafka-clients.version>
+    <spring-kafka.version>1.3.10.RELEASE</spring-kafka.version>
```

Comment:
I am curious, since we are adding this bundle , is there a reason for not going with latest version of the spring-kafka lib (version 2.45)  and kafka-client ( version 2.5)

Reply:
We don't have any specific need other than the fact that we can get the bug fixes/improvements that went in newer version. Just wanted to understand the reason. I am fine with this version if the spring 5 version has issues working in OSGI